stdlib/bigdecimal/0/big_decimal.rbs

```diff
@@ -1187,6 +1187,18 @@ class BigDecimal < Numeric
   #
   def zero?: () -> bool
 
+  # call-seq:
+  #     a.to_d -> bigdecimal
+  #
```

Comment:
Maybe `call-seq` is not necessary.
I think you should use the `rbs annotate` tool.